nexus/src/config.rs

```diff
@@ -330,6 +340,7 @@ mod test {
                         .parse()
                         .unwrap()
                 },
+                updates: UpdatesConfig { tuf_trusted_root: None },
```

Comment:
I think this test fills everything out instead of leaving anything optional.  It seems useful to have at least one test that does that for the updates section.